build_test.go

```diff
@@ -72,80 +74,10 @@ func testBuild(t *testing.T, when spec.G, it spec.S) {
 		tmpDir, err = ioutil.TempDir("", "build-test")
 		h.AssertNil(t, err)
 
-		builderName = "example.com/default/builder:tag"
+		defaultBuilderName = "example.com/default/builder:tag"
 		defaultBuilderStackID = "some.stack.id"
-		defaultBuilderImage = ifakes.NewFakeBuilderImage(t,
```

Comment:
Moved this into a helper function so that I can re-use it to make a builder that contains a version of the lifecycle that supports the creator.